contracts/farm-bot.sol

```diff
@@ -441,6 +448,13 @@ contract FarmBot is ERC20, AccessControl {
             );
         }
         revoFees.issueCompounderBonus(msg.sender);
-        emit ClaimRewards(msg.sender, lpBalance);
+        emit ClaimRewards(
+            msg.sender,
+            lpBalance,
```

Comment:
Sending `lpBalance`, `interestEarnedNumerator`, and `interestEarnedDenominator` is sufficient to reconstruct both the previous and current values of `lpTotalBalance` right?